class.phpmailer.php

```diff
@@ -908,10 +926,14 @@ protected function addAnAddress($kind, $address, $name = '')
             }
             return false;
         }
+
         if ($kind != 'Reply-To') {
             if (!array_key_exists(strtolower($address), $this->all_recipients)) {
                 array_push($this->$kind, array($address, $name));
-                $this->all_recipients[strtolower($address)] = true;
```

Comment:
By not adding "envelope_to" recipients in the $this->all_recipients array, you allow duplicates in the "envelope_to" array. You need to add a mechanism to disallow duplicates in the "envelope_to" array.